### ISSUE TITLE
Profile menu should be a menu, not modal

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/LITEFile#139]
The idea of displaying this status information in a card isn't bad, it's just surprising for the profile button in the top right to pop up a centered modal, and I think is counter to accessibility:

<img width="1682" height="824" alt="Image" src="https://github.com/user-attachments/assets/129d3cdd-348f-4ec4-9e32-959fe092cbde" />

Here's how a few others handle this mixed task of including actions + a card w/ current login information:

<img width="563" height="497" alt="Image" src="https://github.com/user-attachments/assets/0bc44e47-acd6-4d14-9f7a-d9e47f30cda7" />

<img width="774" height="825" alt="Image" src="https://github.com/user-attachments/assets/1f93a204-01d4-46b3-ab4e-c7a493d4e45a" />

<img width="607" height="1133" alt="Image" src="https://github.com/user-attachments/assets/ec5e8611-e39a-491d-b36a-d82d65544fdf" />

We should also consider showing the logged-in user's name here. Not all websites do this, but we have plenty of room and it could add some clarity that there are actions that the user can take (such as changing who is logged in) via this menu button.
